instrumentation/kafka-clients/README.md

```diff
@@ -10,6 +10,7 @@ First, setup the generic Kafka component like this:
 kafkaTracing = KafkaTracing.newBuilder(messagingTracing)
                            .writeB3SingleFormat(true) // for more efficient propagation
                            .remoteServiceName("my-broker")
+                           .singleRootSpanOnReceiveBatch(true) // to share a single root span when starting a trace from consumer
```

Comment:
Since `singleRootSpanOnReceiveBatch` is the default you may omit it there IMO, it's not needed to setup kafkatracing. We may explain the feature better in a dedicated paragraph of README.md to make sure people understand why this feature was added, its impact and when it comes into play. The rationale you wrote in #1039 is a good start.

(not part of this PR but consider then removing the line `writeB3SingleFormat(true)` as well since it's the default and deprecated anyway)